### PR TITLE
fixed pillow import

### PR DESCRIPTION
**Pull request by pmeier**
Corrects #29 

Change the required `pillow` version from `<= 6.2.0` to `< 7.0.0` to include the most recent versions in between.

### CODE REVIEW

File: pystiche/image/io.py [hunk 1]
```diff
@@ -36,6 +36,6 @@ def write_image(
 
 
 def show_image(
-    image: torch.Tensor, mode: Optional[str] = None, title: Optional[str] = None,
+    image: torch.Tensor, mode: Optional[str] = None, title: Optional[str] = None
```
Comment (pmeier) [inline pystiche/image/io.py]: Unrelated change from `black`

**Comment by hugovk** [comment]
[torchvision v0.5.0](https://github.com/pytorch/vision/releases/tag/v0.5.0) has been released with the fix:

1. Require `torchvision>=0.5.0`
2. If Pillow was temporarily pinned, remove the pin
